similarity_search/test/test_distfunc.cc

```diff
@@ -785,8 +785,8 @@ bool TestSparseCosineSimilarityAgree(const string& dataFile, size_t N, size_t Re
 
     bool bug = false;
 
-    float maxRelDiff = 1e-6f;
-    float maxAbsDiff = 1e-6f;
+    float maxRelDiff = 1e-5f;
+    float maxAbsDiff = 1e-5f;
```

Comment:
Yep, these are not really unit tests. But I prefer to have at least such checks for integrity (although some unit testing gurus would disagree).